content/en/docs/distributions/aws/pipeline.md

```diff
@@ -1,7 +1,7 @@
 +++
-title = "Configure Kubeflow Pipelines on AWS"
-description = "Customize Kubeflow Pipelines to use AWS Services"
-weight = 90
+title = "Kubeflow Pipelines on AWS"
+description = "Kubeflow Pipelines on Amazon EKS"
+weight = 50
```

Comment:
I saw those weight are kind of different from each other, should we make it same in this case?

Reply:
I reset some of the weights given the reorganization, and these leave some room for more sections. I think it's ok as long as they come up in a sensible order. But let me know if I'm missing something?